app/views/chapters/show.html.erb

```diff
@@ -6,21 +15,24 @@
   <h1>
     <span class="hidden-xs"><%= t(:chapter_number, number: @chapter.number) %>:&nbsp;</span>
     <span><%= @chapter.name %></span>
+    <%= render_if_mono_lesson partial: 'layouts/guide_title_icons' %>
   </h1>
 
   <%= @chapter.description_html %>
 
 </div>
 
 <% if @chapter.lessons.present? %>
-<div>
-  <h3><%= t(:lessons) %></h3>
+  <%= render_if_mono_lesson partial: 'layouts/mono_lesson', locals: { subject: @lesson } do %>
```

Comment:
It would be nice to add some kind of `mu-monolesson` class in order to be able to customize these paths from custom css.

Reply:
:up: Please don't forget about this renaming and the css classes :pray: :pray:

Reply:
Yesss, I took over this one but was paused because _other stuff_